### PR TITLE
chore: add Aaquib Ali

### PR DESCRIPTION
**Pull request by aaquibdev**
## Changes proposed

Adding links to linkfree in public/data 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/975"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]

File: public/data/aaquibdev.json
Comment (krishguptadev) [inline public/data/aaquibdev.json:40]:
```suggestion

```

**Review by krishguptadev — COMMENTED** [review]
~~And the formatting isn't correct~~ It will be done by actions

**Comment by aaquibdev** [comment]
@krishguptadev  i done the modification check again if you find any problem plesae tell....?

**Review by nathgoutam93 — APPROVED** [review]

**Review by vikasganiga05 — APPROVED** [review]

**Comment by aaquibdev** [comment]
@vikasganiga05   when this Pr is closed..?

**Comment by vikasganiga05** [comment]
> @vikasganiga05 when this Pr is closed..?

I don't know. It depends on the maintainers. They will do a final review and merge this PR. :slightly_smiling_face:

**Comment by emmalearnscode** [comment]
@aaquibdev We usually aim for 2 maintainers to approve the PR before merging :)

**Review by krishguptadev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, Thanks for the contribution :D.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**
